db/seeds.rb

```diff
@@ -1,5 +1,9 @@
 # coding: utf-8
 
+AdminUser.create!(email: 'admin@example.com',
+                  password: 'password',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.